topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -0,0 +1,599 @@
+> ```
+> ---------------------Mate 1--------------------------------->
+> <---------------------Mate 2----------------------
+> ```
+>
+> or
+>
+> ```
+> ---------------------Mate 1--------------------------------->
+> <---------------------Mate 2---------------------------------
+> ```
+>
+> This is what we call dovetailing and we want to consider this pair as a valid concordant alignment.
+{: .comment}
+
+
+> ### {% icon hands_on %} Hands-on: Mapping reads to reference genome
+>
+> 1. {% tool [Bowtie2](toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2/2.4.2+galaxy0) %} with the following parameters:
+>    - *"Is this single or paired library"*: `Paired-end`
+>        - {% icon param-file %} *"FASTQ/A file #1"*: select the output of **Trim Galore!** {% icon tool %} *"Read 1 Output"*
+>        - {% icon param-file %} *"FASTQ/A file #2"*: select the output of **Trim Galore!** {% icon tool %} *"Read 2 Output"*
+>        - *"Do you want to set paired-end options?"*: `Yes`
+>            - *"Set the maximum fragment length for valid paired-end alignments"*: `1000`
+>            - *"Allow mate dovetailing"*: `Yes`
+>    - *"Will you select a reference genome from your history or use a built-in index?"*: `Use a built-in genome index`
+>        - *"Select reference genome"*: `Human (Homo sapiens): hg38 Canonical`
+>    - *"Set read groups information?"*: `Do not set`
+>    - *"Select analysis mode"*: `1: Default setting only`
+>        - *"Do you want to use presets?"*: `Very sensitive end-to-end (--very-sensitive)`
+>    - *"Do you want to tweak SAM/BAM Options?"*: `No`
+>    - *"Save the bowtie2 mapping statistics to the history"*: `Yes`
+>
+> 2. Click on the {% icon galaxy-eye %} (eye) icon of the mapping stats.
+{: .hands_on}
+
+> ### {% icon comment %} Bowtie2 Results
+> You should get similar results to this from Bowtie2:
+> ![Mapping statistics of bowtie2](../../images/cut_and_run/mapping_stats.png "Mapping statistics of bowtie2")
+{: .comment}
+
+> ### {% icon question %} Questions
+>
+> What percentage of read pairs mapped concordantly?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 46.09+52.04=98.13%
+> >
+> {: .solution}
+>
+{: .question}
+
+# Step 3: Filtering Mapped Reads and second level Quality Control
+
+## Filter Uninformative Reads
+
+We apply some filters to the reads after the mapping. CUT & RUN datasets can have many reads that map to the mitchondrial genome because it is nucleosome-free and thus very accessible to the pA-MNase. CUT & RUN inserts adapter more easily in open chromatin regions due to the pA-MNase activity. The mitchondrial genome is uninteresting for CUT&RUN, so we remove these reads. We also remove reads with low mapping quality and reads that are not properly paired.
```

Comment:
```suggestion
We apply some filters to the reads after the mapping. CUT&RUN datasets can have many reads that map to the mitchondrial genome because it is nucleosome-free and thus very accessible to the pA(G)-MNase. CUT&RUN inserts adapter more easily in open chromatin regions due to the pA-MNase activity. The mitchondrial genome is uninteresting for CUT&RUN, so we remove these reads. We also remove reads with low mapping quality and reads that are not properly paired.
```